src/redux/types.js

```diff
@@ -0,0 +1,4 @@
+export const GET_ROCKETS = 'get_rockets';
+export const GET_MISSIONS = 'get_missions';
+export const ROCKET_RESERVED = 'rocket_reserved';
+export const MISSIONS_ACTIONS = 'missions_actions';
```

Comment:
- [OPTIONAL] I would recommend using the duck pattern for your actions.
- [OPTIONAL] Usually action types file is closer to actions, so I would recommend using the same folder for both

Reply:
@EricMbouwe,
Thanks for the feedback.
We will do the optional suggested changes.